README.md

```diff
@@ -5,6 +5,36 @@ ufo-hunters.com
 
 A site that collects a huge data set of UFO sightings all over the world. Our main purpose is to collaborate spreading objective UFO data.
 
+
+Overview
+------------
+
+This site is built with an incredibly powerful web framework called Ruby on Rails, that uses the most eloquent and elegant programming language we've ever used.
+
+UFO-hunters.com was realized from scratch with Ruby 1.9 and Rails 3.3.2 and now upgrade to Ruby 2.1.2 and Rails 4.1.2 .
+
+All UFO data we collect is regularly imported into a NoSQL database called MongoDB, with geospatial "2dsphere" indexes that considerably ease all the stuff with coordinates and geolocation. 
+
+The site allow access to private zone with session variables. The user can to create his own articles.
```

Comment:
Yo pondría: "The site allows access to a private area where the users can create their own articles"